tests/lightning_qubit/test_measurements_class.py

```diff
@@ -480,7 +487,7 @@ def test_single_return_value(self, measurement, observable, lightning_sv, tol):
         assert np.allclose(result, expected, max(tol, 1.0e-4))
 
     @flaky(max_runs=5)
-    @pytest.mark.parametrize("shots", [None, 1000000])
+    @pytest.mark.parametrize("shots", [None, 1000000, (900000, 900000)])
```

Comment:
How's this shots-vector test related to the PR?

Reply:
Yes, I added it because @vincentmr [request](https://github.com/PennyLaneAI/pennylane-lightning/pull/817#discussion_r1695158901) to cover this part of the code.